### ISSUE TITLE
Let's classify jobpostings

### ISSUE THREAD

**Issue by cbleek** [cbleek/JobClassification#1]
Create a CR and put it into the directory [concepts](concepts/)

We offer $50 on [upwork](https://www.upwork.com/ab/applicants/1089625636831813632/job-details) for a concept. We want to have several ideas. Ideas will be public.

So let's share our knowledge.

**Comment by cbleek** [comment]
let's start by predicting the nature of the employment.

Google [defines](https://developers.google.com/search/docs/data-types/job-posting?hl=de)

- "FULL_TIME"
- "PART_TIME"
- "CONTRACTOR"
- "TEMPORARY"
- "INTERN"
- "VOLUNTEER"
- "PER_DIEM"
- "OTHER"

so we should take these categories.

The question is what a process that classifies job ads might look like. The process starts with collecting job ads. Let's assume that we have data, as in the [example folder.](https://github.com/cbleek/JobClassification/tree/master/example-data)

What does an API have to look like to get the data. How can a process that processes the data look like? And how can the process learn?

**Comment by mirfan899** [comment]
Well, category you have given not present in the data. Each job ad must have category in the `examples` directory data files.

**Comment by cbleek** [comment]
Yes, the field does not exist in the [example data](https://github.com/cbleek/JobClassification/tree/master/example-data). So we need a process. An idea of how to solve this problem.

An idea would be to search the source code of a job ad for an ld-json of type "JobPosting".

Example:

https://www.brose.com/de-de/karriere/job-praktikant-fertigungsteam--4503.html contains such a Json 

With job postings in German, you can often tell from the title whether it is an internship or a full-time job. It would be a different approach.

**Comment by mirfan899** [comment]
This needs scrapping the site for possible jobs id and then those job title can be taken as category.

**Comment by mirfan899** [comment]
To classify adds we need to have a dataset, in `examples` there are files related to jobs but format is not consistent in files. Data should be in same format for machine learning to work. Also some jobs have missing values. An important feature for classification is requirements of the job which can be used to categories the job ads.

**Comment by cbleek** [comment]
jobs, located at https://www.jobs.de/suche?utf8=✓&q=&loc= are containing good lpublic d-json data. I hope, that these data will give us good dataset.
